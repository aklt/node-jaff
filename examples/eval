#begin coffee
str = 'Hello string!'
arr = [
  100
  200
  'plain arrays become lines'
  ]

obj =
  foo: [
    bar:
      hat: 100
      cats: 'no'
  ,
    bob: [
      100
      300
    ]
  ,
    flap:
      broo: 'goo'
      schmoo: 'doo'
      glue: 1001
  ,
    hat:
      yes: [ 10, 10, 100, 1000]
  ]

arrObj = [
  100
  'fresh foo'
  obj:
    a: 100
    b: 200
 ,
  hats:
    g: 'big hat'
    c: 'caps'
 ,
  'Foo'
]
#end
string:
{str}
object:
{obj}
array:
{arr}
array with objects:
{arrObj}
