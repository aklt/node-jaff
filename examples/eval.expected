string:
Hello string!
object:
{ foo: 
   [ { bar: { hat: 100, cats: 'no' } },
     { bob: [ 100, 300 ] },
     { flap: { broo: 'goo', schmoo: 'doo', glue: 1001 } },
     { hat: { yes: [ 10, 10, 100, 1000 ] } } ] }
array:
100
200
plain arrays become lines
array with objects:
100
fresh foo
{ obj: { a: 100, b: 200 } }
{ hats: { g: 'big hat', c: 'caps' } }
Foo
